### DISCUSSION TITLE
1.3.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#200] [Обновления]
### Список изменений
- Исправлена работа для сайтов Piped
- Уменьшен отступ кнопки перевода от верхнего края видео
- Убрана ненужная проверка в регексе на фрагмент "/clip/" для Twitch (она не работало, потому что клипы на твиче имеют вид https://www.twitch.tv/КАНАЛ/clip/АЙДИ_КЛИПА)
- Добавлена поддержка Vimeo Direct Links (https://vimeo.com/ХХХ/ХХХ, если Яндекс сделают для них поддержку, то они будут работать, пока висим на "видео не доступно для перевода")
- Добавлена поддержка Twitch Clips (https://www.twitch.tv/NICKNAME/clip/)
- Добавлена поддержка Twitch Clips Embedded (https://clips.twitch.tv/embed)
- Добавлена поддержка VK Видео Embedded (https://vk.com/video_ext.php)
- Возвращена поддержка YouTube Embedded (https://www.youtube.com/embed/XXX)
- Добавлены минифицированные версии расширения (находятся в папке dist)

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.3.0...1.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.3.1'>1.31.</a>.</em>

**Comment by CrescentXIII** [comment]
Спасибо за вашу работу!
Есть небольшое неудобство, в Опере, когда в фуллэкране, слишком высоко плашка расширения, и курсор просто не может на нее нажимать, т.к. вверху перекрывает поле с названием ролика... (не знаю, как показать, скрин может поможет).

UPD: добавлю, возможно будет важно, это при масштабировании системы 150% (1080p) на ТВ используется ПК.
![изображение_2023-06-30_185050508](https://github.com/ilyhalight/voice-over-translation/assets/130666598/247bbcba-ca44-4381-8f30-edae1f5a5711)

**Reply by ilyhalight** [reply]
Как-то не замечал этого, ибо всегда использовал кнопку до перехода в полный экран. Наверное, верну как было

**Reply by CrescentXIII** [reply]
Ну, хотел бы внести предложение маленькое еще. Если это другим не будет мешать, то можно опустить "кнопку" ниже всплывающего окошка когда разворачиваешь на весь экран? (см. скрин)  Раньше кнопка как раз за этим сообщением была (накладывалась), приходится ждать секунды 3 пока не исчезнет оно. Спасибо 
![image](https://github.com/ilyhalight/voice-over-translation/assets/130666598/37c09b16-4e71-463b-b8b6-61acbf63479d)
